### PR TITLE
Pass comma-separated couchdb servers to haproxy

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: scripts/build/cht-compose.yml.template [hunk 1]
```diff
@@ -57,17 +57,11 @@ services:
     image: {{{ repo }}}/cht-haproxy:{{ tag }}
     container_name: {{ haproxy_container_name }}
     hostname: haproxy
-    depends_on:
-      - couchdb.1
-      - couchdb.2
-      - couchdb.3
```
Comment (garethbowen) [inline scripts/build/cht-compose.yml.template:63]: I don't think removing this is necessary, is it?

What's the advantage of starting haproxy before the couch images have started?

File: haproxy/haproxy.cfg [hunk 2]
```diff
@@ -64,6 +64,3 @@ backend couchdb-servers
   retry-on all-retryable-errors
   log global
   retries 5
-  server couchdb1 $COUCHDB1_SERVER:5984 check inter 2s
```
Comment (garethbowen) [inline haproxy/haproxy.cfg:67]: Add a comment or something here to explain how the couchdb servers get appended.

**Review by garethbowen — COMMENTED** [review]
Nice. A couple of suggestions.

File: scripts/build/cht-compose.yml.template (hunk 1 above)
Comment (dianabarsan) [reply scripts/build/cht-compose.yml.template:63]: Well, with the configurable containers, we no longer know if these containers (and services) even exist. :(

Comment (garethbowen) [reply scripts/build/cht-compose.yml.template:63]: But they're defined in this file above?

Comment (dianabarsan) [reply scripts/build/cht-compose.yml.template:63]: They're just temporarily there, I'm splitting them out in the next PR, as per our discussion.

Comment (garethbowen) [reply scripts/build/cht-compose.yml.template:63]: Coolio.

**Review by garethbowen — APPROVED** [review]
LGTM! Thanks.

**Pull request merged**